TOOLING.md

```diff
@@ -0,0 +1,15 @@
+# Tooling
+
+This documents describes at a high level some the key tools used across `near-js`.
+
+## Package Manager - [PNPM](https://pnpm.io/)
+PNPM is a package manager focussed on fast installations while still be disk efficient. It manages dependency installations across the monorepo as well as linking interdependent packages. Unlike other package managers, PNPM has a non-flat `node_modules` structure which ensures packages only have access to dependencies defined in their `package.json`.
+
+## Build System - [Turborepo](https://turborepo.org/)
+Turborepo provides tooling for orchestrating commands across the monorepo. Similar to Yarn Workspaces, Turbo can run tasks within each project via a single command, but rather than executing serially, Turbo has various performance enhancing features such as task parallelization and local/remote caching.
+
+## Package Publishing - [Changesets](https://github.com/changesets/changesets)
+Changesets manages package versioning, changelog generation, and publishing to the package registry. Changesets provides a CLI to simplify bumping packages and adding changelog entries, it then automates versioning and publishing via Github Actions.
```

Comment:
manages -> manage
provides -> provide
Github -> GitHub (apparently this is their official naming convention)

Reply:
Updated to `GitHub` but the others don't seem correct to me?

Reply:
I'm guessing that grammar checker considered `Changesets` as plural. (It is not, or is it?) if it is not, then ignore my comment as technically `manages` and `provides` aren't wrong.

Reply:
Yeah I think you're right. "Changesets" isn't plural so I'll leave as is :)